src/helpers/BoxHelper.js

```diff
@@ -101,5 +101,20 @@ BoxHelper.prototype.setFromObject = function ( object ) {
 
 };
 
+BoxHelper.prototype.copy = function ( source ) {
+
+	LineSegments.prototype.copy.call( this, source );
+
+	this.object = source.object;
+
+	return this;
+
+};
+
+BoxHelper.prototype.clone = function () {
```

Comment:
Since the `clone()` method is inherited, it does not need to be redefined here. However, it doesn't hurt, and I think it makes the code easier to follow. I do not object to removing it though.